docs/docs/scanner/misconfiguration/config/config.md

```diff
@@ -0,0 +1,262 @@
+This page describes misconfiguration-specific configuration.
+
+### External connectivity
```

Comment:
this section is repeated several times in other places. maybe we can skip it here?